### PR TITLE
Add org.gtk.Gtk3theme.Chicago95

### PR DESCRIPTION
**Pull request by adamthiede**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I have [built][build] and tested the submission locally.
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. Still waiting on response: https://github.com/grassmunk/Chicago95/issues/106
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[maint]: https://docs.flathub.org/docs/for-app-authors/maintanance
[submission]: https://docs.flathub.org/docs/for-app-authors/submission
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: org.gtk.Gtk3theme.Chicago95.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="runtime">
+  <id>org.gtk.Gtk3theme.Chicago95</id>
+  <metadata_license>GPL-3.0</metadata_license>
+  <name>Chicago95 Gtk Theme</name>
+  <summary>A rendition of everyone's favorite 1995 Microsoft operating system for Linux.</summary>
```
Comment (hfiguiere) [inline org.gtk.Gtk3theme.Chicago95.appdata.xml:6]:
```suggestion
  <summary>A rendition of everyone's favorite 1995 Microsoft operating system for Linux</summary>
```

**Comment by adamthiede** [comment]
Upstream seems to be a loosely coordinated cohort of contributors, and the deb/rpm maintainer said "go for it". I'll of course gladly hand over reins if someone from there wants to take this up.

**Comment by adamthiede** [comment]
bot, build org.gtk.Gtk3teme.Chicago95

**Comment by adamthiede** [comment]
bot, build org.gtk.Gtk3theme.Chicago95

**Comment by grassmunk** [comment]
Thanks @adamthiede, not entirely sure what next steps I need to make but I've accepted the invite.

**Comment by adamthiede** [comment]
Nothing really necessary; if you have flatpaks you can just install the theme from flathub and it'll work now. If you ever want to make any changes in that repo don't feel like you need to ask me; I just wanted to get it out there.

**Pull request closed without merging**